dayPlanner.html

```diff
@@ -17,72 +17,73 @@
               </tr>
-              <tr onclick="{clickableTR5()}">
+              <tr onclick="{funcGroup.func5()}">
                 <th>07:00 - 07:30</th>
-                <td></td>
+                <td id="td5"></td>
               </tr>
-              <tr onclick="{clickableTR6()}">
+              <tr onclick="{funcGroup.func6()}">
                 <th>07:30 - 08:00</th>
-                <td></td>
+                <td id="td6"></td>
               </tr>
-              <tr onclick="{clickableTR7()}">
+              <tr onclick="{funcGroup.func7()}">
                 <th>08:00 - 08:30</th>
-                <td></td>
+                <td id="td7"></td>
               </tr>
-              <tr onclick="{clickableTR8()}">
+              <tr onclick="{funcGroup.func8()}">
                 <th>08:30 - 09:00</th>
-                <td></td>
+                <td id="td8"></td>
               </tr>
-              <tr onclick="{clickableTR9()}">
+              <tr onclick="{funcGroup.func9()}">
                 <th>09:00 - 09:30</th>
-                <td></td>
+                <td id="td9"></td>
               </tr>
-              <tr onclick="{clickableTR10()}">
+              <tr onclick="{funcGroup.func10()}">
                 <th>09:30 - 10:00</th>
-                <td></td>
+                <td id="td10"></td>
               </tr>
-              <tr onclick="{clickableTR11()}">
+              <tr onclick="{funcGroup.func11()}">
                 <th>10:00 - 10:30</th>
-                <td></td>
+                <td id="td11"></td>
               </tr>
-              <tr onclick="{clickableTR12()}">
+              <tr onclick="{funcGroup.func12()}">
                 <th>10:30 - 11:00</th>
-                <td></td>
+                <td id="td12"></td>
               </tr>
-              <tr onclick="{clickableTR13()}">
+              <tr onclick="{funcGroup.func13()}">
                 <th>11:00 - 11:30</th>
-                <td></td>
+                <td id="td13"></td>
               </tr>
-              <tr onclick="{clickableTR14()}">
+              <tr onclick="{funcGroup.func14()}">
                 <th>11:30 - 12:00</th>
-                <td></td>
+                <td id="td14"></td>
               </tr>
-              <tr onclick="{clickableTR15()}">
+              <tr onclick="{funcGroup.func15()}">
                 <th>12:00 - 12:30</th>
-                <td></td>
+                <td id="td15"></td>
               </tr>
             </tbody>
           </table>
           <div id="settingBox">
             <div class="settingBoxContent">
               <span id="close">&times;</span>
-              <p>Lorem ipsum, dolor sit amet consectetur adipisicing elit. Odio at commodi ea, quibusdam, soluta excepturi veritatis possimus quisquam minus obcaecati vel incidunt? Assumenda voluptatem sint soluta veniam quo animi sit!</p>
+              <input id="todoArea">
+              <button id="addTodo">EKLE</button>

```

Comment:
- input area'daki metni `tr` targetına yerleştirebilmek için bir buton yerleştiriyoruz